.travis.yml

```diff
@@ -9,3 +9,7 @@ deploy:
   domain: 42six
   on:
     repo: 42sixsolutions/claire
+
+before_install:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
```

Comment:
karma launches firefox to test, and travis can't launch firefox without this yml change